draft-ietf-rats-eat.md

```diff
@@ -225,6 +225,23 @@ An EAT may be encoded in either JavaScript Object Notation (JSON) {{RFC8259}} or
 EAT is built on CBOR Web Token (CWT) {{RFC8392}} and JSON Web Token (JWT) {{RFC7519}} and inherits all their characteristics and their security mechanisms.
 Like CWT and JWT, EAT does not imply any message flow.
 
+Following is a very simple example.
+It is JSON format for easy reading, but could also be CBOR.
+Only the Claims-Set, the payload for the JWT, is shown.
+
+~~~~
+{::include cddl/Example-Payloads/simple.json}
+~~~~
+
+This example has a nonce for freshness.
+This nonce is the base64 encoding of a 12 byte random binary byte string.
+The ueid is effectively a serial number uniquely identifying the device.
+This ueid is the base64 encoding of a 48-bit MAC address.
```

Comment:
PR looks good. One minor comment. It may be worth calling out the type byte. Perhaps: "This ueid is the base64 encoding of a 48-bit MAC address along with a type byte set to 0x02."

Reply:
maybe it's also s good case for using the term `base64url-encoded` from the document glossary, as in:
```
This nonce is the base64url-encoded representation of a 12 byte random binary byte string.
```